en/components/overlay_main.md

```diff
@@ -40,11 +34,11 @@ export class MyOverlayComponent {
 
 The overlay service can be used to dynamically display a `HTMLNode` or even an Angular Component by attaching it to the overlay DOM.
 
-After a reference to the Overlay service is established, it can be used to dynamically show/hide content. For example, we can pass an Angular Component in the `attach` function. This will generate unique ID. We can pass then this ID to `show` function to display the component:
+After a reference to the Overlay service is established, it can be used to dynamically show/hide content. For example, we can pass an Angular Component in the `attach` function. This will generate a unique ID, which we can pass to the `show` function to display the component:
 
 ```typescript
 
-// in my-overlay-component.component.ts
+// overlay-component.component.ts
```

Comment:
if we change the name we should change it and in the next line like this:
```
import { DynamicComponent } from '../dynamic-component/dynamic-component.component';
```

Reply:
what does it have to do with the name of the file?